<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fso::fred::dialogs::ShipEditorDialog</class>
 <widget class="QDialog" name="fso::fred::dialogs::ShipEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Ship</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="prevButton">
             <property name="text">
              <string>Prev</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="weaponsButton">
           <property name="text">
            <string>Weapons</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playerOrdersButton">
           <property name="text">
            <string>Player Orders</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="specialExpButton">
           <property name="text">
            <string>Special Exp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="specialHitsButton">
           <property name="text">
            <string>Special Hits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Wing:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Hotkey</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Kill Score</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="hotkeyCombo"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Static</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="killScoreEdit"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="personaCombo"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Persona</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Assist %</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="assistEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="playerShipCheckBox">
             <property name="text">
              <string>Player Ship</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="playerShipButton">
           <property name="text">
            <string>Set As Player Ship</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="altShipClassButton">
           <property name="text">
            <string>Alt Ship Class</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Team</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="shipClassCombo"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Alt Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>AI Class</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="altNameCombo">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Ship Class</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="teamCombo"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ship Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="AIClassCombo"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cargoCombo">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="shipNameEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Cargo</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Callsign</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="callsignCombo">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="textureReplacementButton">
           <property name="text">
            <string>Texture Replacement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="miscButton">
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initialStatusButton">
         <property name="text">
          <string>Initial Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initialOrdersButton">
         <property name="text">
          <string>Initial Orders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tblInfoButton">
         <property name="text">
          <string>TBL Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideCuesButton">
         <property name="text">
          <string>Hide Cues</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="arrivalGroupBox">
         <property name="title">
          <string>Arrival</string>
         </property>
         <layout class="QVBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Distance</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Location</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="arrivalTargetCombo"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Target</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Delay</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="arrivalLocationCombo"/>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QSpinBox" name="arrivalDelaySpinBox"/>
              </item>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Seconds</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="arrivalDistanceEdit"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="restrictArrivalPathsButton">
            <property name="text">
             <string>Restrict Arrival Paths</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="customWarpinButton">
            <property name="text">
             <string>Custom Warp-in Parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Cue:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="updateCueCheckBox">
              <property name="text">
               <string>Update Cue</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="fso::fred::sexp_tree" name="arrivalTree"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="departureGroupBox">
         <property name="title">
          <string>Departure</string>
         </property>
         <layout class="QVBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Target</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="departureLocationCombo"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Delay</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Location</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QSpinBox" name="departureDelaySpinBox"/>
              </item>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Seconds</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="departureTargetCombo"/>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6"/>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>19</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="restrictDeparturePathsButton">
            <property name="text">
             <string>Restrict Departure Paths</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="customWarpinButton">
            <property name="text">
             <string>Custom Warp-out Parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Cue:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="updateCueCheckBox">
              <property name="text">
               <string>Update Cue</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="fso::fred::sexp_tree" name="departureTree"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>fso::fred::sexp_tree</class>
   <extends>QTreeView</extends>
   <header>ui/widgets/sexp_tree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
